katas/content/multi_qubit_systems/prepare_superposition/index.md

```diff
@@ -1,8 +1,8 @@
 **Input:** A two-qubit system in the basis state $|00\rangle = \begin{bmatrix} 1 \\ 0 \\ 0 \\ 0 \end{bmatrix}$.
 
-**Goal:** Transform the system into the state $\frac{1}{\sqrt2}\big(|00\rangle - |01\rangle\big) = \frac{1}{\sqrt2}\begin{bmatrix} 1 \\\ -1 \\\ 0 \\\ 0 \end{bmatrix}$.
+**Goal:** Transform the system into the state $\frac{1}{\sqrt2}\big(|00\rangle - |10\rangle\big) = \frac{1}{\sqrt2}\begin{bmatrix} 1 \\\ 0 \\\ -1 \\\ 0 \end{bmatrix}$.
```

Comment:
```suggestion
**Goal:** Transform the system into the state $\frac{1}{\sqrt2}\big(|00\rangle - |10\rangle\big) = \frac{1}{\sqrt2}\begin{bmatrix} 1 \\\ 0 \\\ -1 \\\ 0 \end{bmatrix}$.

**Note:** Note that the quantum states are in little-endian (i.e. $|q_1 q_0\rangle$).
```

Reply:
Could you please add this note to all the other exercises?

Reply:
I took some inspiration from your note in one of the solutions.

Reply:
I've removed the note from solutions and added your note to the descriptions.